compute/src/test/resources/initscript_with_java.sh

```diff
@@ -177,7 +177,11 @@ function installOpenJDK() {
     echo reusing JAVA_HOME $JAVA_HOME
   else
     if which dpkg &> /dev/null; then
-      apt-get-update && apt-get-install openjdk-6-jdk
+      apt-get update && \
+          PACKAGE=`apt-cache search --names-only '^openjdk-.-jdk$' | sort -r | cut -d' ' -f1 | head -1` && \
+          [ ! -z "$PACKAGE" ] && \
+          DEBIAN_FRONTEND=noninteractive && \
+          {  apt-get -q -y install $PACKAGE-headless || apt-get -q -y install $PACKAGE; }
```

Comment:
Better use the [apt-get-install](https://github.com/jclouds/jclouds/blob/master/scriptbuilder/src/main/resources/functions/setupPublicCurl.sh#L18) alias.